frontend/language/src/nb.json

```diff
@@ -951,6 +951,11 @@
   "schema_editor.maxLength": "Maksimal lengde",
   "schema_editor.maximum_exclusive": "Mindre enn ",
   "schema_editor.maximum_inclusive": "Mindre enn eller lik",
+  "schema_editor.metadata": "Metadata",
+  "schema_editor.metadata.autoCreate": "Opprett automatisk",
+  "schema_editor.metadata.maxCount": "Maksimum antall",
+  "schema_editor.metadata.minCount": "Minimum antall",
+  "schema_editor.metadata.not_found": "Finner ikke metadata",
   "schema_editor.minItems": "Minimalt antall elementer",
   "schema_editor.minLength": "Minimal lengde",
```

Comment:
```suggestion
  "schema_editor.minLength": "Minste mulige lengde",
```

Reply:
Hvis du ønsker å beholde "minumum" og "maksimum" @Jondyr , så skriv det over alt, ikke bruk minimal(t). Da blir det for eksempel "minimumslengde og maksimumslengde.